rotkehlchen/exchanges/binance.py

```diff
@@ -338,12 +339,17 @@ def api_query(
 
             api_subdomain = api_type if is_new_futures_api else 'api'
             request_url = (
-                f'https://{api_subdomain}.{self.uri}{api_type}/v{api_version!s}/{method}?'
+                f'https://{api_subdomain}.{self.uri}{api_type}/v{api_version!s}/{method}'
             )
-            request_url += urlencode(call_options)
             log.debug(f'{self.name} API request', request_url=request_url)
             try:
-                response = self.session.get(request_url, timeout=timeout)
+                if request_type == 'get':
+                    response = self.session.get(
+                        url=request_url + f'?{urlencode(call_options)}',
+                        timeout=timeout,
```

Comment:
instead of concatenating to the url you can use

```suggestion
                        url=request_url,
                        params=call_options,
                        timeout=timeout,
```

check it but I believe it should work